src/main/java/seedu/foodrem/model/item/ItemQuantity.java

```diff
@@ -2,9 +2,7 @@
 
 import static java.util.Objects.requireNonNull;
 
-import java.util.function.BiFunction;
-
-import seedu.foodrem.model.item.itemvalidator.ItemQuantityValidator;
+import seedu.foodrem.model.item.itemvalidators.ItemQuantityValidator;
```

Comment:
My bad, my merge resolution caused a compile error. Let's fix this:

```suggestion
import java.util.function.BiFunction;

import seedu.foodrem.model.item.itemvalidators.ItemQuantityValidator;
```